rfc/rfc-0014-change-tracking.md

```diff
@@ -0,0 +1,46 @@
+# RFC 0014 Change Tracking
+
+## Summary
+Implement a change tracking feature to make it possible for clients to track input data as it traverses through the supply chain.
+Motivation
+Context
+At the core of the architecture of cartographer is the concept that a supply chain is the orchestration of objects and their controllers.  
```

Comment:
nit: My understanding is that the core of cartographer is actually `choreography` versus `orchestration` as stated here.

Reply:
Changed.